gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/util/ParticipantReferenceUtil.java

```diff
@@ -51,6 +54,21 @@ private static Optional<String> getParticipantReference(List<RCMRMT030101UK04Par
             .findFirst();
     }
 
+    public static Reference getParticipant2Reference(RCMRMT030101UK04EhrComposition ehrComposition, String typeCode) {
```

Comment:
Could we refactor this and `getParticipant2Reference` to reduce the duplication?